mdtraj/core/trajectory.py

```diff
@@ -162,20 +162,20 @@ def _parse_topology(top):
         ext = None  # might not be a string
 
     if isinstance(top, string_types) and (ext in ['.pdb', '.pdb.gz', '.h5','.lh5']):
-        _traj = load_frame(top, 0)
+        _traj = load_frame(top, 0, **kwargs)
         topology = _traj.topology
     elif isinstance(top, string_types) and (ext in ['.prmtop', '.parm7']):
-        topology = load_prmtop(top)
+        topology = load_prmtop(top, **kwargs)
     elif isinstance(top, string_types) and (ext in ['.psf']):
-        topology = load_psf(top)
+        topology = load_psf(top, **kwargs)
     elif isinstance(top, string_types) and (ext in ['.mol2']):
-        topology = load_mol2(top).topology
+        topology = load_mol2(top, **kwargs).topology
     elif isinstance(top, string_types) and (ext in ['.gro']):
-        topology = load_gro(top).topology
+        topology = load_gro(top, **kwargs).topology
     elif isinstance(top, string_types) and (ext in ['.arc']):
-        topology = load_arc(top).topology
+        topology = load_arc(top, **kwargs).topology
     elif isinstance(top, string_types) and (ext in ['.hoomdxml']):
-        topology = load_hoomdxml(top).topology
+        topology = load_hoomdxml(top, **kwargs).topology
     elif isinstance(top, Trajectory):
         topology = top.topology
```

Comment:
In this branch, `kwargs` is ignored.  In every other routine, if `kwargs` contains an unrecognized/unsupported argument, an exception is thrown.  Should I add something here like

```Python
if kwargs:
    raise TypeError('Unrecognized arguments %s' % ', '.join(kwargs.keys()))
```

for consistency?